_data/legal.yml

```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Clio
+      url: https://www.clio.com
+      img: clio.png
+      tfa: Yes
+      software: Yes
```

Comment:
Removed email, added software. 2FA is provided through Google Authenticator (actually, anything that supports T-OTP).